src/sagemaker/workflow/_repack_model.py

```diff
@@ -34,7 +34,7 @@
 from distutils.dir_util import copy_tree
 
 
-def repack(inference_script, model_archive, dependencies=None, source_dir=None):
+def repack(inference_script, model_archive, dependencies=None, source_dir=None):  # pragma: no cover
```

Comment:
why we want to mark this function as "no cover"?

Reply:
This function is internally used and only used in a script to repack model. It does have corresponding [unit test file](https://github.com/aws/sagemaker-python-sdk/blob/dev/tests/unit/sagemaker/workflow/test_repack_model_script.py) but all of them are skipped due to 
```
This test operates on the root file system and will likely fail due to permission errors.
```

Therefore, I guess we don't need this calculated in the coverage report